src/client/pages/Confirmation/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Helmet from 'react-helmet-async'
+import { TopBar } from 'components/TopBar'
+import { Page } from 'components/utils/Page'
+import { SessionTokenGuard } from 'containers/SessionTokenGuard'
+import { useTextKeys } from 'utils/textKeys'
+import { DownloadPageContent } from '../Download/components/DownloadPageContent'
+
+export const Confirmation = () => {
+  const textKeys = useTextKeys()
+
+  return (
+    <Page>
+      <SessionTokenGuard>
+        <>
+          <Helmet>
+            <title>{textKeys.CONFIRMATION_PAGE_TITLE()}</title>
+          </Helmet>
```

Comment:
I guess we don't need this if we handle it serverside, right?

Reply:
Yeah, I guess `app.use(helmet)` takes care of that on the server..? We can even remove `<title>` since the server renders it for us!